src/components/custom/lastest-news.tsx

```
@@ -72,7 +74,7 @@ export default function WynncraftNews() {
                   <p className="text-sm text-muted-foreground mb-2">
                     {formatDate(item.date)} by {item.author}
                   </p>
-                  <p className="text-sm mb-2">{truncateContent(item.content, 150)}</p>
+                  <p className="text-sm mb-2" dangerouslySetInnerHTML={{__html: truncateContent(item.content, 150)}}></p>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)